components/CounterForm.jsx

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import { useCart } from "../context/ItemsInCart";
+
+const CounterForm = ({ slug, name, isTiny }) => {
+  const { itemsInCart, setItemsInCart, addItem } = useCart();
+  const [amount, setAmount] = useState(0);
+
+  const decAmount = (int) => {
+    setAmount((amount) => amount - int);
+  };
+
+  const incAmount = (int) => {
+    setAmount((amount) => amount + int);
+  };
```

Comment:
I'd try to resist the urge to use abbr. wherever possible it makes your c. harder to follow:

```suggestion
  const decrementAmount = (by) => {
    setAmount((amount) => amount - by);
  };

  const incrementAmount = (by) => {
    setAmount((amount) => amount + by);
  };
```